src/typescript/azure-openapi-validator/rules/utilities/resourceUtils.ts

```diff
@@ -195,6 +199,42 @@ export class ResourceUtils {
     }
     return topLevelModels
   }
+
+  public getTopLevelResourcesByRG() {
+    const fullModels = this.getOperationGetResponseResources()
+    const topLevelModels = new Set<string>()
+    for (const entry of fullModels.entries()) {
+      const paths = entry[1]
```

Comment:
Could you pls add comments here? Why '1'? looks like a magic number

Reply:
The entries() return  k-v pair array which contains 2 items , not magic number